securityconfig/audit.yml.example

```diff
@@ -0,0 +1,52 @@
+_meta:
+  type: "audit"
+  config_version: 2
+
+config:
+  # enable/disable auditlog
+  enabled: true
+
+  audit:
+    # rest
+    enable_rest: true
+    disabled_rest_categories:
+      - AUTHENTICATED
+      - GRANTED_PRIVILEGES
+
+    # transport
+    enable_transport: true
+    disabled_transport_categories:
+      - AUTHENTICATED
+      - GRANTED_PRIVILEGES
+
+    # ignore
+    ignore_users:
+      - kibanaserver
+    ignore_requests: []
+
+    # verbose attributes
+    resolve_bulk_requests: false
+    log_request_body: true
+    resolve_indices: true
+    exclude_sensitive_headers: true
+
+  compliance:
+    # enable/disable compliance
+    enabled: true
+
+    # configs
+    internal_config: true
+    external_config: false
+
+    # compliance read
+    read_metadata_only: true
+    read_watched_fields: {}
```

Comment:
read_watched_fields is going to be a map because fields are involved. 
```
read_watched_fields : {
  "indexName1": ["field1", "field2"],
  "indexPattern": ["fields"]
}
```
whereas write_watched_indices is just list of indexes. 
```
write_watched_indices: ["indexName1", "indexPattern"]
```

Reply:
Thanks. It is best to add sample/ comment in example YML file